src/apigateway/explorer/apiGatewayNodes.ts

```diff
@@ -23,7 +23,7 @@ import globals from '../../shared/extensionGlobals'
 export class ApiGatewayNode extends AWSTreeNodeBase {
     private readonly apiNodes: Map<string, RestApiNode>
 
-    public constructor(private readonly partitionId: string, private readonly regionCode: string) {
+    public constructor(private readonly partitionId: string, public readonly regionCode: string) {
```

Comment:
@JadenSimon if we don't want to put a region field on AWSTreeNodeBase then we could create `AWSTreeNodeBase2` which does have a region field. Though I do not see much harm in adding a field that might be irrelevant (and undefined) for some nodes?

Reply:
oh I see we already have `RegionNode`, which I think is the subclass I was imagining.

Reply:
Adding more to `AWSTreeNodeBase` is ok IMO because the vast majority of the uses are for the AWS explorer. It's not really a generic tree node class.